### ISSUE TITLE
Control all Displays from CLI

### ISSUE THREAD

**Issue by alexmerm** [alin23/Lunar#377]
### Name of the feature

Change Properties of all Monitors with CLI

### What problem does this feature solve?

Right now in the CLI (as far as I can tell), the "set" and "get" subcommands can only operate on the first active display, making it impossible to change the brightness, contrast, etc. of any other displays from the CLI. 

### How should the feature work
Add an option to the "set" and "get" commands to specify the display number. For example: 
```
lunar set --display 1 brightness 75
lunar set --display all contrast 3 
```

**Comment by alin23** [comment]
@alexmerm `lunar set/get` are mostly thought as shortcuts for when you have a single monitor.

For more complex cases where you have more than one monitor, `lunar displays` can be used like so:

```sh
# It's best to use the serial to identify a monitor as that is the only value that is guaranteed to be persistent

# SET: lunar displays <serial of the display> <property to change> <value>
lunar displays CF4CF870-1DB2-B8D5-0857-D6964E3302DB brightness 100

# GET: lunar displays <serial of the display> <property to change>
lunar displays CF4CF870-1DB2-B8D5-0857-D6964E3302DB brightness
```

I also added the possibility to use `all` for setting values and I'll release that in the next version:

```sh
lunar displays all brightness 100
```

**Comment by alexmerm** [comment]
Whoops didn't realize that. Thanks sm for adding the all displays feature. Maybe you could also change the "displays" entry in `lunar --help` , right now it just says ` Lists currently active displays` which isn't exactly clear that all that functionality is in there.

**Comment by alin23** [comment]
Fixed in 4.5.1
